### ISSUE TITLE
Support manually scraped sites

### ISSUE THREAD

**Issue by HarelM** [IsraelHikingMap/Site#713]
## Feature
Some of the data that we would like to present in our site is data that is not accessible through API calls.
This data can be scraped out and added to database or files manually.
Once added it should be displayed in our site much like any other 3rd party data.

**Comment by HarelM** [comment]
Eretz HaMaayanot, converted to the required format.
[eMaayanot.csv.txt](https://github.com/IsraelHikingMap/Site/files/2087043/eMaayanot.csv.txt)

**Comment by HarelM** [comment]
@valleyofdawn
The following header (order is not important) is expected in order for this to work properly on the site.
I wrote some code to convert the original csv to the expected csv (some information is not straight forward such as ID and geo location, and other stuff I added manually to the csv file).
`Id,Title,Description,Website,ImageUrl,Category,FileUrl,Icon,IconColor,Latitude,Longitude,SourceImageUrl`
Explanation on each column can be found in this file:
https://github.com/IsraelHikingMap/Site/blob/07efbdf7105e7d5f7a465c2903aa55dd314dc2df/IsraelHiking.API/Services/Poi/CsvPointsOfInterestAdapter.cs#L22

This is mainly for future reference and documnetation, I was able to make Eretz Hamaayanot work locally on my PC :-).
